Terminal.Gui/Views/TextField.cs

```diff
@@ -40,7 +40,7 @@ public class TextField : View {
 		/// Gets or sets the foreground <see cref="Color"/> to use when 
 		/// rendering <see cref="Caption"/>.
 		/// </summary>
-		public Color CaptionColor { get; set; } = Color.DarkGray;
+		public Color CaptionColor { get; set; } = (Color)Color.DarkGray;
```

Comment:
I love that it is possible to cast to `Color` from the enum but for readability it might be better in our code to use the constructor directly (even though they are functionally the same).

I think the code is more readable when you have:

```csharp
public Color CaptionColor { get; set; } = new Color(ColorNames.DarkGray);
```

Reply:
I think it's 6 one way, a half-dozen the other. But I love consistency, so I'm going to follow your suggestion and make all our code use `new`.